mbed_greentea/mbed_test_api.py

```diff
@@ -140,7 +143,10 @@ def run_htrun(cmd, verbose):
             gt_logger.gt_log_err(test_error.group(1))
 
         if verbose:
-            sys.stdout.write(line.decode('utf-8').rstrip() + '\n')
+            try:
+                sys.stdout.write(line.decode('utf-8').rstrip() + '\n')
```

Comment:
can we replace `os.linesep` instead of `\n`

Reply:
I agree. We should file a different PR for that